### PR TITLE
boards/stm32l476g-disco: enable cpy2remed

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
### Contribution description

This PR enables `cpy2remed` programmer for `stm32l476g-disco` board. Moreover, it adds documentation page for this boards.

### Testing procedure

Go to `blinky` program and flash boards using command:

```
make BOARD=stm32l476g-disco PROGRAMMER=cpy2remed flash
```

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__stm32l476g-disco.html
```

### Issues/PRs references

None

### CODE REVIEW

File: boards/stm32l476g-disco/Makefile.include [hunk 1]
```diff
@@ -3,11 +3,16 @@ INCLUDES += -I$(RIOTBOARD)/common/stm32/include
...
 # use connect_assert_srst to always be able to flash or reset the board.
 OPENOCD_RESET_USE_CONNECT_ASSERT_SRST ?= 1
 
-# openocd programmer is supported
-PROGRAMMERS_SUPPORTED += openocd
+# openocd and cpy2remed programmer is supported
+PROGRAMMERS_SUPPORTED += openocd cpy2remed
+
+#needed by cpy2remed
+FLASHFILE += $(HEXFILE)
```
Comment (benpicco) [inline boards/stm32l476g-disco/Makefile.include:17]: `makefiles/tools/cpy2remed.inc.mk` already takes care of that

File: boards/stm32l476g-disco/Makefile.include [hunk 2]
```diff
@@ -3,11 +3,16 @@ INCLUDES += -I$(RIOTBOARD)/common/stm32/include
 
 # this board uses openocd with st-link
 PROGRAMMER ?= openocd
+
```
Comment (benpicco) [inline boards/stm32l476g-disco/Makefile.include:6]:
```suggestion
```

File: boards/stm32l476g-disco/Makefile.include (hunk 1 above)
Comment (krzysztof-cabaj) [reply boards/stm32l476g-disco/Makefile.include]: Fixed.

File: boards/stm32l476g-disco/Makefile.include (hunk 2 above)
Comment (krzysztof-cabaj) [reply boards/stm32l476g-disco/Makefile.include]: Fixed.

**Review by benpicco — APPROVED** [review]

File: boards/stm32l476g-disco/doc.txt [hunk 3]
```diff
@@ -0,0 +1,60 @@
+/**
+ * @defgroup    boards_stm32l476g-disco STM32L476G-DISCO
```
[the remark this answers is not in the document]
Comment (krzysztof-cabaj) [reply boards/stm32l476g-disco/doc.txt:2]: I change `board.h` - changes in `doc.txt` did not work - doc page contains only links to other files.

File: boards/stm32l476g-disco/include/board.h [hunk 4]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    boards_stm32l476g-disco STM32L476G-DISCO
+ * @ingroup    boards_stm32l476g-disco STM32L476G-DISCO
  * @ingroup     boards
```
Comment (benpicco) [inline boards/stm32l476g-disco/include/board.h:11]:
```suggestion
```

File: boards/stm32l476g-disco/include/board.h [hunk 5]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    boards_stm32l476g-disco STM32L476G-DISCO
+ * @ingroup    boards_stm32l476g-disco STM32L476G-DISCO
```
Comment (benpicco) [inline boards/stm32l476g-disco/include/board.h:10]:
```suggestion
 * @ingroup    boards_stm32l476g-disco
```

Comment (krzysztof-cabaj) [reply boards/stm32l476g-disco/include/board.h:10]: Ok. Maybe this resolve the problem with static tests ;) - but now doc page looks good.

**Comment by krzysztof-cabaj** [comment]
The CI is happy (all green) page now looks good (in my humble opinion). 

@benpicco thanks for help with doxygen groups.

**Comment by benpicco** [comment]
Thank you for your doc improvements!

**Pull request merged**